scipy/linalg/tests/test_lapack.py

```diff
@@ -1786,6 +1794,165 @@ def test_getc2_gesc2():
                                       x/scale, decimal=4)
 
 
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_gttrf_gttrs(dtype):
+    # The test uses ?gttrf and ?gttrs to solve a random system for each dtype,
+    # tests that the output of ?gttrf define LU matricies, that input
+    # parameters are unmodified, transposal options function correctly, that
+    # incompatible matrix shapes raise an error, and singular matrices return
+    # non zero info.
+
+    np.random.seed(42)
+    n = 10
+    rtol = 250 * np.finfo(dtype).eps  # set test tolerance appropriate for dtype
```

Comment:
Rtol is not needed here since atol is same order of magnitude due random arrays. You can set it to 0.

Reply:
Before changing it, I remember @mdhaber spent some time thinking through the tolerances. @mdhaber  do you have any thoughts on using rtol vs atol?

Reply:
Hmm, I don't remember. @ilayn's suggestion makes sense to me, so let's try that.

Reply:
@Kai-Striega changed this.